src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,182 @@
+                updateListOfTask(command, tasks);
+            }
+        }
+    }
+
+    private static void printWelcomeMessage() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+        printSingleLine();
+        System.out.println("Hello! I'm duke.Duke");
+        System.out.println("How can I help you?");
+        printSingleLine();
+    }
+
+    private static void printGoodbyeMessage() {
+        System.out.println("Bye. See you soon!");
+        printSingleLine();
+    }
+
+    private static void printListOfTask(Task[] tasks) {
+        if (Task.getNumberOfTask() == 0) {
+            System.out.println("Wow! I see that your list is empty");
+        } else {
+            System.out.println("Here are the tasks in your list:");
+            for (int i = 0; i < Task.getNumberOfTask(); i++) {
+                System.out.println((i + 1) + "." + tasks[(i + 1)].toString());
+            }
+        }
+        printSingleLine();
+    }
+
+    private static void printDetailsOfAddedTask(Task newTask) {
+        System.out.println("One more thing you got to do. Press on!");
+        System.out.println(newTask.toString());
+        System.out.println("Now you have " + Task.getNumberOfTask() + " tasks in the list.");
+        printSingleLine();
+    }
+
+    private static void updateTaskAsComplete(Task[] tasks, String description) {
+        int taskNumber = Integer.parseInt(description);
+        tasks[taskNumber].markAsDone();
+        printDetailsOfCompletedTask(tasks[taskNumber]);
+    }
+
+    private static void printDetailsOfCompletedTask(Task task) {
+        System.out.println("Nice. One more down!");
+        System.out.println(task.toString());
+        printSingleLine();
+    }
+
+    private static void updateListOfTask(String command, Task[] tasks) {
+        int indexToSplit = command.indexOf(' ');
+        if (indexToSplit == -1) {
+            printInvalidTaskMessage();
+        } else {
+            String typeOfTask = command.substring(0, indexToSplit);
+            String description = command.substring(indexToSplit);
+            description = description.trim();
+
+            determineTypeOfTask(tasks, typeOfTask, description);
+        }
+    }
+
+    private static void determineTypeOfTask(Task[] tasks, String typeOfTask, String description) {
+        switch (typeOfTask) {
+        case "todo":
+            createTodoTask(tasks, description);
+            break;
+        case "deadline":
+            checkValidityOfDeadline(tasks, description);
+            break;
+        case "event":
+            checkValidityOfEvent(tasks, description);
+            break;
+        case "done":
+            checkValidityOfCompletedTask(tasks, description);
+            break;
+        default:
```

Comment:
Nice alignment of the case labels to the switch statement according to the standard.